packages/client/test/e2e.spec.ts

```diff
@@ -0,0 +1,141 @@
+/*
+  This test script (e2e.spec.ts) aims to perform integrated testing of the project. It executes a series of actions,
+  including deploying the contracts (registry, offchain resolver, and universal resolver), creating the Client using Viem, 
+  and initializing the gateway locally. After deploying and configuring the contracts, the Client can access
+  off-chain information during the tests. It's important to note that this initial test script only sets up the
+  environment and stops at the gateway call. It still requires implementing the connection between the gateway and 
+  layer two, or the gateway and the database.
+*/
+
+// Importing abi and bytecode from contracts folder
+import {
+  abi as abiOffchainResolver,
+  bytecode as bytecodeOffchainResolver,
+} from '../../contracts/out/OffchainResolver.sol/OffchainResolver.json'
```

Comment:
I'm pretty sure you could use

```suggestion
} from '@blockful/contracts/out/OffchainResolver.sol/OffchainResolver.json'
```

but you'd need to add the following as client's `peerDependencies`

```
"@blockful/contracts": "workspace:^"
````

Reply:
did it! nice catch.